app/javascript/entrypoints/application.js

```diff
@@ -68,18 +68,3 @@ window.log_plausible_connect_orcid = function () {
 /* eslint-enable no-console */
 /* eslint-enable no-undef */
 /* eslint-enable func-names */
-
-/* eslint-disable no-restricted-globals */
-/* eslint-disable no-alert */
-window.confirmOrcidRemove = function confirmOrcidRemove(id) {
```

Comment:
No need to prompt the user anymore since the new functionality will only expire tokens that are indeed not valid (whereas in the previous version we were expiring *all* tokens so it made sense to check with the user first)